quad_utils/src/quad_kd.cpp

```diff
@@ -7,7 +7,7 @@ Eigen::IOFormat CleanFmt(4, 0, ", ", "\n", "[", "]");
 QuadKD::QuadKD()
 {
   std::string robot_description_string;
-  if (!ros::param::get("robot_description", robot_description_string))
+  if (!ros::param::get("/robot_description", robot_description_string))
```

Comment:
See above comment, although it might be fine to leave this as checking the global namespace for now.